wrangler-service/src/main/java/co/cask/wrangler/service/spanner/SpannerSpecification.java

```diff
@@ -29,10 +29,10 @@
   private final String instance;
   private final String database;
   private final String table;
-  private final Schema schema;
+  private final String schema;
 
   SpannerSpecification(String serviceFilePath, String project, String instance,
-                       String database, String table, Schema schema) {
+                       String database, String table, String schema) {
```

Comment:
lets still pass in a Schema, but set the field using toString(). This is to make sure it's always set to a valid value.